nano/core_test/message.cpp

```diff
@@ -243,6 +243,29 @@ TEST (message, confirm_req_hash_batch_serialization)
 	ASSERT_EQ (header.count_get (), req.roots_hashes.size ());
 }
 
+TEST (message, confirm_req_to_string)
```

Comment:
I asked for unit tests and thank you for doing them but let's make them minimal.
It is not worth having and maintaining large amount of code testing to_string functions.